src/components/emby-button/emby-button.css

```diff
@@ -29,6 +29,13 @@
     text-decoration: none;
     /* Not crazy about this but it normalizes heights between anchors and buttons */
     line-height: 1.35;
+    transform-origin: center;
+    transition: 0.2s;
+}
+
+.emby-button.show-focus:focus {
+    transform: scale(1.4);
```

Comment:
isn't it getting a bit too big?

Reply:
That scale was the default value before, I just moved it here using the new method so we can more easily search and modify TV layout changes in the future.